Athos/ONNXCompiler/ONNXNodesAST.py

```diff
@@ -283,6 +283,28 @@ def ConstantOfShape(node, value_info, node_name_to_out_var_dict, innermost_let_a
 
 		return (innermost_let_ast_node, out_var_count)	
 
+	def Pad(node, value_info, node_name_to_out_var_dict, innermost_let_ast_node, out_var_count, mtdAST):

```

Comment:
Great! I was worried that without the refactoring this might be impossible to understand.

Reply:
life finds a way :)